src/status_im/subs.cljs

```diff
@@ -1633,3 +1634,27 @@
  :<- [:search/filter]
  (fn [[chats search-filter]]
    (apply-filter search-filter chats extract-chat-attributes)))
+
+;;ENS ==================================================================================================================
+
+(re-frame/reg-sub
+ :ens/state
+ (fn [db]
```

Comment:
for root keys we need to use `reg-root-key-sub`

Reply:
and clearly here  there should be only one root key for ens and the rest is derived subscription. but looking at how they are used it should even rather be screen subscriptions that aggregate from an :ens root key and a few others

Reply:
Not clear what you mean. Do you have an exemple?

Reply:
https://github.com/status-im/status-react/blob/develop/src/status_im/subs.cljs#L1199-1212 is a good exemple

Basically you need to think about each subscription you make as a cache, in your case you want to cache the :ens key of app-db as a root key, so that the subscriptions you use it in are only recomputed when the something in :ens is changed.